vendor/CMakeLists.txt

```diff
@@ -26,6 +30,13 @@ if(EXISTS "${ANDROID_PATCH_DIR}/")
 	endforeach(v)
 endif()
 
+find_package(Protobuf REQUIRED)
```

Comment:
If protobuf is now required, please also add it to the `Dependencies` section in `README.md` :pray:

Reply:
I'll add it into `README.md`